app/src/main/java/org/gdg/frisbee/android/common/GdgNavDrawerActivity.java

```diff
@@ -134,15 +139,17 @@ private void setupDrawerContent(NavigationView navigationView) {
         }
 
         menu.add(GAMES_GROUP_ID, Const.DRAWER_ACHIEVEMENTS, Menu.NONE, R.string.achievements).setIcon(R.drawable.ic_drawer_achievements);
-        menu.add(GAMES_GROUP_ID, Const.DRAWER_ARROW, Menu.NONE, R.string.arrow).setIcon(R.drawable.ic_drawer_arrow);
+        menu.add(GAMES_GROUP_ID, Const.DRAWER_ARROW, Menu.NONE, R.string.arrow).setIcon(R.drawable.ic_drawer_arrow).setCheckable(true);
 
         menu.add(SETTINGS_GROUP_ID, Const.DRAWER_SETTINGS, Menu.NONE, R.string.settings).setIcon(R.drawable.ic_drawer_settings);
         menu.add(SETTINGS_GROUP_ID, Const.DRAWER_HELP, Menu.NONE, R.string.help).setIcon(R.drawable.ic_drawer_help);
         menu.add(SETTINGS_GROUP_ID, Const.DRAWER_FEEDBACK, Menu.NONE, R.string.feedback).setIcon(R.drawable.ic_drawer_feedback);
         menu.add(SETTINGS_GROUP_ID, Const.DRAWER_ABOUT, Menu.NONE, R.string.about).setIcon(R.drawable.ic_drawer_about);
 
-        navigationView.setNavigationItemSelectedListener(
+        // Enables the navigationView to "check" the items of this group when selected
+        menu.setGroupCheckable(GROUP_ID, true, true);
```

Comment:
@friedger you mean the behavior of the GROUP? So far I only added checkable to the top group and the Arrow element. All the other ones in the settings should remain unchecked as well the one that pops up the achievements activity. 

@tasomaniac I've did tests last night and worked as expected.